api/src/routes/api/mock-sas9.ts

```diff
@@ -0,0 +1,145 @@
+import { readFile } from '@sasjs/utils'
+import express from 'express'
+import path from 'path'
+
+const mockSas9Router = express.Router()
+
+const { MOCK_SERVERTYPE } = process.env
+
+let loggedIn: boolean = false
+
+mockSas9Router.get('/SASStoredProcess', async (req, res) => {
+  if (!loggedIn) {
+    res.redirect('/SASLogon/login')
+    return
+  }
+
+  const filePath = path.join(
+    process.cwd(),
+    'mocks',
+    'generic',
+    'sas9',
+    'sas-stored-process'
+  )
+
+  let file
+
+  try {
+    file = await readFile(filePath)
+  } catch (err: any) {
+    console.error(`Mocked file on path: ${filePath} is not found.`)
+    res.status(403).send(err.toString())
```

Comment:
we can do `return res.status(403)...` instead of returning empty on next line

Reply:
Refactor was done after this review, can you please take a quick look again?